spec/components/utility/work_history_component_spec.rb

```diff
@@ -22,28 +22,26 @@
   context 'with work experiences' do
```

Comment:
![image](https://user-images.githubusercontent.com/42515961/135258995-cdd10c72-1edd-4b30-9757-2bcdb9504a25.png)

In the candidate interface we use this component the flag is off